ui/pages/settings/networks-tab/networks-tab-content/networks-tab-content.test.js

```diff
@@ -3,22 +3,14 @@ import configureMockStore from 'redux-mock-store';
 import { fireEvent, screen } from '@testing-library/react';
 import { renderWithProvider } from '../../../../../test/jest/rendering';
 import { defaultNetworksData } from '../networks-tab.constants';
+import { mockNetworkState } from '../../../../../test/stub/networks';
+import { CHAIN_IDS } from '../../../../../shared/constants/network';
 import NetworksTabContent from '.';
 
 const mockState = {
   metamask: {
-    providerConfig: {
-      chainId: '0x539',
-      nickname: '',
-      rpcPrefs: {},
-      rpcUrl: 'http://localhost:8545',
-      ticker: 'ETH',
-      type: 'localhost',
-    },
-    orderedNetworkList: {
-      chainId: '0x539',
-      rpcUrl: 'http://localhost:8545',
-    },
+    ...mockNetworkState({ chainId: CHAIN_IDS.LOCALHOST }),
+    orderedNetworkList: {},
```

Comment:
Should `orderedNetworkList` still include entries for the used chain?

Reply:
ah yeah, i put it back to the way it was before in https://github.com/MetaMask/metamask-extension/pull/26150/commits/059ee02788cac2848847bb12f0dbb5094c578563